swagger_server/utils/db_utils.py

```diff
@@ -1,64 +1,51 @@
 import logging
 import os
 
-import _pickle as pickle
-import dataset
+import pymongo
 
+DB_NAME = os.environ.get("DB_NAME")
```

Comment:
What does `DB_NAME` mean here?  It would be useful to have some docstrings.

Reply:
It's the db_name in mongo_client.list_database_names(). I feel this variable is just a standard db term that's used across most databases...